src/plugins/intel_cpu/src/nodes/def_conv.cpp

```diff
@@ -867,7 +867,7 @@ void DeformableConvolution::initSupportedPrimitiveDescriptors() {
         enforceRef = false;
     }
 
-    if (enforceRef) {
+    if (enforceRef) {  // NOLINT(bugprone-branch-clone)
```

Comment:
I guess instead of disabling the warning, the final 'else' branch can be removed, and impl_type can be initialized with 'ref' in scope of declaration.

Reply:
Done